index.d.ts

```diff
@@ -0,0 +1,27 @@
+export = launchWebAuthFlow;
+
+declare function launchWebAuthFlow(options: WebAuthFlowOptions): ResponseUrl;
```

Comment:
Is there a way to tell typescript that this function returns a promise? i.e. this is an async function.

Reply:
This can be done with `Promise<ResponseUrl>` I'll fix it in the next patch